### DISCUSSION TITLE
Where does the uplift tree n_reg regularization come from?

### DISCUSSION THREAD

**Discussion by brrm** [uber/causalml#690] [Q&A]
I was wondering where exactly the `n_reg` regularization parameter (in the `UpliftTreeClassifier` and `UpliftRandomForestClassifier` classes) was defined? 
The documentation cites Rzepakowski et al. 2012. 
I assume this is refering to _Decision trees for uplift modeling with single and multiple treatments_, but as far as I can tell that paper does not mention regularization anywhere.
Thanks!

**Comment by jeongyoonlee** [comment]
cc: @zhenyuz0500 @t-tte
